src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const meetsCondition = (string: string) => string
```

Comment:
Consider renaming 'meetsCondition' to a more descriptive name, such as 'matchesSearchQuery'.